--- a/requirements-test.txt
+++ b/requirements-test.txt
@@ -1,5 +1,5 @@
-coverage==4.4.1
-idna==2.5
-pytest==2.9.2
-pytest-cov==2.3.0
-tox==2.6.0
+coverage
+idna
+pytest
+pytest-cov
+tox
--- a/tox.ini
+++ b/tox.ini
@@ -4,16 +4,16 @@
 [testenv]
 changedir = .tox
 deps = -rrequirements-test.txt
-commands = coverage run --parallel --rcfile {toxinidir}/.tox-coveragerc -m pytest --doctest-modules {envsitepackagesdir}/hyperlink {posargs}
+commands = python3-coverage run --parallel --rcfile {toxinidir}/.tox-coveragerc -m pytest --doctest-modules {envsitepackagesdir}/hyperlink {posargs}
 
 # Uses default basepython otherwise reporting doesn't work on Travis where
 # Python 3.6 is only available in 3.6 jobs.
 [testenv:coverage-report]
 changedir = .tox
 deps = coverage
-commands = coverage combine --rcfile {toxinidir}/.tox-coveragerc
-           coverage report --rcfile {toxinidir}/.tox-coveragerc
-           coverage html --rcfile {toxinidir}/.tox-coveragerc -d {toxinidir}/htmlcov
+commands = python3-coverage combine --rcfile {toxinidir}/.tox-coveragerc
+           python3-coverage report --rcfile {toxinidir}/.tox-coveragerc
+           python3-coverage html --rcfile {toxinidir}/.tox-coveragerc -d {toxinidir}/htmlcov
 
 
 [testenv:packaging]
